Launcher/Classes/ApplicationStateHandler.swift

```diff
@@ -15,11 +15,13 @@ class ApplicationStateHandler {
         case debugState = "debugCheckboxState"
         case cucumberProfile = "cucumberProfile"
         case additionalRunParameters = "additionalRunParameters"
+        case deviceIP = "deviceIP"
+        case bundleID = "bundleID"
     }
     
-    var simulatorRadioButtonState: Int {
+    var physicalRadioButtonState: Bool {
```

Comment:
This doesn't seem right. You removed `simulatorRadioButtonState`, and the renamed `physicalRadioButtonState` references `.simulatorRadioButtonState`.